recipes/keychain/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 11 CXX_STANDARD_REQUIRED ON)
```

Comment:
if this is required you are missing 

https://github.com/conan-io/conan-center-index/blob/8d389aa0c380f1ea7d0cb9324768ad6051c04874/recipes/elfio/all/conanfile.py#L19-L21

Reply:
fixed, thanks!